gdx/src/com/badlogic/gdx/graphics/g2d/TextureAtlas.java

```diff
@@ -544,7 +570,8 @@ public AtlasRegion (TextureRegion region) {
 		}
 
 		@Override
-		/** Flips the region, adjusting the offset so the image appears to be flip as if no whitespace has been removed for packing. */
+		/** Flips the region, adjusting the offset so the image appears to be flip as if no whitespace has been removed for

```

Comment:
Typo? I think `appears to be flip` should be `appears to be flipped` or  `appears flipped` ?